build.gradle

```
@@ -2,6 +2,8 @@
 buildscript {
     ext {
         kotlin_version = '1.7.10'
+        // KSP releases need to match the Kotlin version - https://github.com/google/ksp/releases
+        ksp_version = '1.7.10-1.0.6'
```

Since the versions need to match, should we do this?

```suggestion
        ksp_version = "$kotlin_version-1.0.6"
```